strapi/src/api/problem/content-types/problem/schema.json

```
@@ -17,8 +18,35 @@
     "description": {
       "type": "richtext"
     },
-    "model_code": {
-      "type": "text"
+    "difficulty": {
+      "type": "string"
+    },
```

Difficulty は、選択式にできたりしますか？
現段階では `'Easy' | 'Medium' | 'Hard'` の 3 種類でしたっけ？

---

こちらは残念ながらstrapi の仕様になるので、できない認識です


https://docs.strapi.io/dev-docs/backend-customization/models#model-attributes

---

String types の `enumeration` を使って今やってみたのですが、できそうな感じだったのですが…
```suggestion
    "difficulty": {
      "type": "enumeration",
      "enum": [
        "Easy",
        "Medium",
        "Hard"
      ]
    },
```

---

String types の `enumeration` を使って今やってみたのですが、できそうな感じだったのですが…
```suggestion
    "difficulty": {
      "type": "enumeration",
      "enum": [
        "Easy",
        "Medium",
        "Hard"
      ]
    },
```

---

ありがとうございます

enumerationがあったんですね
見逃してました。。。

---

[b48ed30](https://github.com/Greek-Academy/alpha-dojo-v2/pull/31/commits/b48ed303c5ce9f6f54741f713efa9e0531762a72)で修正済み